### ISSUE TITLE
Questões sobre função

### ISSUE THREAD

**Issue by felipefoschiera** [nycolexavier/doubt#1]
Olá Nycole,
Antes de tentar lhe ajudar a refatorar seu código, tenho algumas questões sobre coisas que parecem estar faltando, ou contexto que é necessário para poder te ajudar. Nunca mexi com Haxe antes, mas acho que entendi um pouco a ideia.
Algumas perguntas para clarificar:

#### 5) Qual é o contexto em que esta função será utilizada?

**Comment by nycolexavier** [comment]
**Olá, Felipe!**

_1_- É uma fração do código de um jogo. Essa função no jogo trocaria as posições dos personagens aleatóriamente depois do usuário acertar, se errar aparece o botão para jogar novamente. 

_2_- `iRepet` e `iOrdemLength`, foram definidas no começo do código assim: `private var iRepet:Int = 0;` e `private var iOrdemLenght:Int = 0;`

_3_- Isso, `aJanelas` é um array definida assim: `private var aJanelas:Array<Dynamic>;`  e assim

`aJanelas = SfArray.getChildsByName(mcCena, 'r', function funF(obj:Object)
		{
			cast(obj, MovieClip).gotoAndStop(1);
			obj.fixo = false;
			evt.add(obj, MouseEvent.MOUSE_DOWN, funClick);
		});
`

_4_- Vish, essa vou ficar te devendo 😬 ainda estou tentando entender esse código também.

_5_- Creio que a primeira responda essa pergunta mas, se não explico mais detalhadamente

**Desde já, muito obrigada!!**

**Comment by felipefoschiera** [comment]
Certo, então creio que há mais coisa além desse código que você tem no repositório (`doubt/Tela1Clica.hx`). Você consegue fazer o upload do resto? Por exemplo, você disse que declarou `iRepet` e `iOrdemLength`, mas no código que está acessível no repositório não há isso.
